api/src/main/resources/Reference_Application_Concepts-23.xml

```diff
@@ -231,123 +239,123 @@
   <concept_name concept_id="1067" name="Desconocido" locale="es" creator="1" date_created="2012-03-04 18:59:44.0" concept_name_id="108335" voided="false" voided_by="1" uuid="108335BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
   <concept_name concept_id="1067" name="Desconocida" locale="es" creator="1" date_created="2012-03-04 18:59:44.0" concept_name_id="108336" voided="false" voided_by="1" void_reason="" uuid="108336BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" locale_preferred="false"/>
   <concept_name concept_id="1067" name="HAIJULIKANI" locale="sw" creator="1" date_created="2012-08-10 19:19:26.0" concept_name_id="110114" voided="false" voided_by="1" uuid="110114BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
-  <concept_name concept_id="1067" name="&#1053;&#1077; &#1080;&#1079;&#1074;&#1077;&#1089;&#1090;&#1085;&#1086;" locale="ru" creator="1" date_created="2013-01-27 01:36:57.0" concept_name_id="111191" voided="false" voided_by="1" uuid="111191BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
-  <concept_name concept_id="1067" name="&#2460;&#2494;&#2472;&#2495;&#2472;&#2494;" locale="bn" creator="1" date_created="2015-04-24 17:19:36.0" concept_name_id="127334" voided="false" voided_by="1" uuid="127334BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
+  <concept_name concept_id="1067" name="Не известно" locale="ru" creator="1" date_created="2013-01-27 01:36:57.0" concept_name_id="111191" voided="false" voided_by="1" uuid="111191BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
+  <concept_name concept_id="1067" name="জানিনা" locale="bn" creator="1" date_created="2015-04-24 17:19:36.0" concept_name_id="127334" voided="false" voided_by="1" uuid="127334BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
   <concept_name concept_id="1067" name="Enkoni" locale="ht" creator="1" date_created="2016-02-20 05:13:35.0" concept_name_id="135594" voided="false" voided_by="1" uuid="135594BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
-  <concept_name concept_id="1067" name="&#1605;&#1593;&#1604;&#1608;&#1605; &#1606;&#1729;&#1740;&#1722;" locale="ur" creator="1" date_created="2016-08-16 07:46:02.0" concept_name_id="139801" voided="false" voided_by="1" uuid="139801BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
+  <concept_name concept_id="1067" name="معلوم نہیں" locale="ur" creator="1" date_created="2016-08-16 07:46:02.0" concept_name_id="139801" voided="false" voided_by="1" uuid="139801BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
```

Comment:
Did you intentionally make changes to this file?

Reply:
I just copied the output of the export tool. I guess chrome maybe converted it automatically?

Reply:
Do we need to change anything in this file?